data_extraction/data_source.yaml

```diff
@@ -3,7 +3,7 @@ morgenstern2024compact:
   is_csv: True
 
 lee2024compact:
-  url: "https://arxiv.org/src/2311.13681"
+  url: "https://arxiv.org/src/2311.13681v1"
```

Comment:
@paulbzm can you review this? I think we'd like to keep the version-less arxiv links, so that they update automatically with new uploads, right?